### PR TITLE
Chain the sharedfx and sharedhost MSIs into a single dotnet redist bundle.

### PR DESCRIPTION
**Pull request by Sridhar-MS**
cc @ellismg @Petermarcu

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/1941)

### CODE REVIEW

**Comment by ellismg** [comment]
I really wish we would segment the packing folder by component and then os, not OS and component.

File: packaging/windows/sharedframework/bundle.wxl [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WixLocalization Culture="en-us" Language="1033" xmlns="http://schemas.microsoft.com/wix/2006/localization">
+  <String Id="Caption">[WixBundleName] Setup</String>
+  <String Id="Title">Microsoft Dotnet CLI for Windows</String>
```
Comment (ellismg) [inline packaging/windows/sharedframework/bundle.wxl]: .NET not dotnet

**Comment by Sridhar-MS** [comment]
My idea was to keep similar scripts together. That makes for better segregation. Also OSX scripts were previously clubbed together.

**Comment by piotrpMSFT** [comment]
We took on a new pivot in the packaging scripts that added to pre-existing source code organization issues. Let's get things building, then reconsider how the source is organized.

**Comment by piotrpMSFT** [comment]
rebased this atop rel/1.0.0 to get rid of known build breaks.

@Sridhar-MS FYI

**Comment by Sridhar-MS** [comment]
Thanks @piotrpMSFT

File: packaging/windows/sharedframework/bundle.wxl (hunk 1 above)
Comment (Sridhar-MS) [inline packaging/windows/sharedframework/bundle.wxl]: Fixed

File: packaging/windows/sharedframework/bundle.wxl [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WixLocalization Culture="en-us" Language="1033" xmlns="http://schemas.microsoft.com/wix/2006/localization">
+  <String Id="Caption">[WixBundleName] Setup</String>
+  <String Id="Title">Microsoft .Net CLI for Windows</String>
+  <String Id="Motto">You just need a shell, a text editor and 10 minutes of your time.
+
+Ready? Set? Let's go!</String>
```
Comment (krwq) [inline packaging/windows/sharedframework/bundle.wxl]: truly interactive

**Comment by Sridhar-MS** [comment]
OSX Jenkins slave went offline during the build - http://dotnet-ci.cloudapp.net/job/dotnet_cli/job/rel_1.0.0/job/release_osx_x64_prtest/337/console

cc: @mmitche

**Comment by mmitche** [comment]
@dotnet-bot test OSX x64 Release Build

**Comment by piotrpMSFT** [comment]
@Sridhar-MS can we merge?

**Comment by Sridhar-MS** [comment]
Yes.

**Pull request merged**